ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml

```diff
@@ -377,35 +377,21 @@ tests:
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
     env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
```

Comment:
@JoelSpeed 
per my understanding, 
the 
```yaml
      BASE_DOMAIN: qe.devcluster.openshift.com
      COMPUTE_NODE_TYPE: m6g.xlarge
      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
      E2E_RUN_TAGS: '@arm64 and @aws-ipi and @disconnected and @network-ovnkubernetes
        and not @fips'
      OCP_ARCH: arm64
      TAG_VERSION: '@4.12'
      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```
should not be deleted.

you only need to add `FEATURE_SET: TechPreviewNoUpgrade` when changing `cucushift-installer-rehearse-aws-ipi-disconnected-private-techpreview` to `cucushift-installer-rehearse-aws-ipi-disconnected-private`

so, the env part should be
```yaml
      BASE_DOMAIN: qe.devcluster.openshift.com
      COMPUTE_NODE_TYPE: m6g.xlarge
      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
      E2E_RUN_TAGS: '@arm64 and @aws-ipi and @disconnected and @network-ovnkubernetes
        and not @fips'
      FEATURE_SET: TechPreviewNoUpgrade
      OCP_ARCH: arm64
      TAG_VERSION: '@4.12'
      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```
adding @aleskandro @liangxia @jianlinliu to correct me if I am wrong.

Reply:
This was a mistake, you're right, it should only have added the new env var, not removed those old ones, this needs to be reverted. Not sure why this happened, perhaps some find/replace didn't work correctly here

Reply:
no need to revert the change, we are trying to fix it in https://github.com/openshift/release/pull/39002